config/model/mixnet/mixnet-m-detection.yaml

```diff
@@ -79,6 +79,14 @@ model:
         # postprocessor - nms
         nms_thresh: 0.45
         class_agnostic: False
```

Comment:
Duplicated lines should be removed.

Reply:
Thank you for your suggestion! I missed them. Please let me check and make commits again!